src/index.js

```diff
@@ -7,9 +7,9 @@
 /**
  * External dependencies
  */
+import 'react-native-get-random-values'; // This library works as a polyfill for the global crypto.getRandomValues which is needed by `uuid` version 7.0.0
```

Comment:
@marecar3 - what is this actually used for? Why is it needed for this feature?
On iOS after removing this line everything seems to still work just fine 🤔

Reply:
For WPiOS we would need to add this extra library too, but I'd like to avoid that if it's not really needed there.

Reply:
This is 3rd party solution that fixes red screen on Android application.